lib/rosette/integrations/smartling-integration/translation_memory.rb

```diff
@@ -0,0 +1,25 @@
+# encoding: UTF-8
+
+module Rosette
+  module Integrations
+    class SmartlingIntegration < Integration
+      class TranslationMemory
+
+        DEFAULT_HASH = {}.freeze
+
+        def initialize(translation_hash)
+          @translation_hash = translation_hash
+        end
+
+        def translation_for(locale, meta_key)
+          all_for_locale = @translation_hash.fetch(locale.code, DEFAULT_HASH)
+
+          if translation = all_for_locale[meta_key]
+            translation.key
+          end
```

Comment:
what will this return if there is no translation? maybe we should do: `all_for_locale[meta_key].try(:key)`?